sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -1232,11 +1226,47 @@ public void testWriteUnknown() throws Exception {
         .withoutValidation());
 
     thrown.expect(RuntimeException.class);
-    thrown.expectMessage("Failed to poll the load job status");
-    p.run();
+    thrown.expectMessage("UNKNOWN status of load job");
```

Comment:
load job id is random
We can refactor the code, and mock it. But, I don't see much value here.